homeassistant/components/osoenergy/water_heater.py

```diff
@@ -46,14 +58,116 @@ async def async_setup_entry(
         return
     async_add_entities((OSOEnergyWaterHeater(osoenergy, dev) for dev in devices), True)
 
+    platform = entity_platform.async_get_current_platform()
+
+    platform.async_register_entity_service(
+        SERVICE_GET_PROFILE,
+        {},
```

Comment:
https://developers.home-assistant.io/blog/2024/08/27/entity-service-schema-validation

Reply:
@joostlek I'm not sure what change you expected here. The changes in the latest commit were not required, please revert them @osohotwateriot

Reply:
I have reverted the changes to the service schema.

Reply:
I've updated the blog post to make it clear passing in a dictionary instead of a schema is still allowed: https://github.com/home-assistant/developers.home-assistant/pull/2316